deployment/templates/curator.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: logging-curator-template-maker
+  annotations:
+    description: "Template to create template for deploying curator"
+    tags: "infrastructure"
+objects:
+- apiVersion: "v1"
+  kind: "Template"
+  metadata:
+    name: logging-${CURATOR_DEPLOY_NAME}-template
+    annotations:
+      description: "Template for logging curator deployment."
+      tags: "infrastructure"
+    labels:
+      logging-infra: curator
+  labels:
+    logging-infra: curator
+    provider: openshift
+    component: ${CURATOR_DEPLOY_NAME}
+  objects:
+  -
+    apiVersion: v1
+    kind: "DeploymentConfig"
+    metadata:
+      name: "logging-${CURATOR_DEPLOY_NAME}"
+      labels:
+        provider: openshift
+        component: "${CURATOR_DEPLOY_NAME}"
+    spec:
+      replicas: 1
+      selector:
+        provider: openshift
+        component: "${CURATOR_DEPLOY_NAME}"
+      triggers:
+      - type: ConfigChange
+      - type: ImageChange
+        imageChangeParams:
+          automatic: true
+          containerNames:
+          - curator
+          from:
+            kind: ImageStreamTag
+            name: logging-curator:latest
```

Comment:
This is only going to exist if you've created the ImageStream, which looks to be missing - should be added to deployment/templates/support.yaml I think for deploying from official registry.